src/policy.rs

```diff
@@ -73,35 +83,34 @@ impl Display for Policy {
 }
 
 impl Policy {
+    /// Generates a new policy object with the given number of attribute
+    /// creation (revocation + addition) allowed.
     #[must_use]
-    pub fn new(nb_revocation: u32) -> Self {
+    pub fn new(nb_creations: u32) -> Self {
         Self {
             last_attribute_value: 0,
-            max_attribute_value: nb_revocation,
-            store: HashMap::new(),
+            max_attribute_value: nb_creations,
+            axes: HashMap::new(),
             attribute_to_int: HashMap::new(),
         }
     }
 
-    /// Returns the policy in the for of a Map where
+    /// Returns the policy in the form of a Map where
     ///  - the keys are the axis names
     ///  - the values are a tuple of
     ///     - list of attribute names for that axis
     ///     - whether the axis hierarchical
     pub fn as_map(&self) -> &HashMap<String, (Vec<String>, bool)> {
-        &self.store
+        &self.axes
     }
 
+    /// Return the number of attribute creations allowed.
     #[must_use]
     pub fn max_attr(&self) -> u32 {
         self.max_attribute_value
     }
```

Comment:
if we rename `max_attribute_value` as suggested, just make it `pub`.

Also, what do you think about adding a new method: `pub fn remaining_attribute_creations(&self)`?

Reply:
this can be done :)